packages/ncids-js/src/components/usa-header/nci-header.component.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Interface for options for the Header component.
+ *
+ * @typedef NCIHeaderOptions
+ * @prop {string} headerOptions Dummy prop for init/testing
+ */
+
+export interface NCIHeaderOptions {
+	headerOptions?: string;
+}
+
+export class NCIHeader {
```

Comment:
So what is your plan for supporting all the different types of headers? Will they be derrived classes?